addons/variable-manager/userscript.js

```diff
@@ -9,6 +9,22 @@ export default async function ({ addon, global, console, msg }) {
   const manager = document.createElement("div");
   manager.classList.add(addon.tab.scratchClass("asset-panel_wrapper"), "sa-var-manager");
 
+  const searchBox = document.createElement('input')
+  searchBox.placeholder = 'search' // todo translation stuff
+  searchBox.style.backgroundImage = `url("${addon.self.dir}/search.svg")` // sad way of doing it but i dont think i can acess addon.self.dir inside of the css but this works so whatever
```

Comment:
Calls to `url()` in non-theme CSS (like this case) are relative to the CSS file's URL, so you don't need to do this from JS.